provider/pkg/provider/provider.go

```diff
@@ -316,13 +316,13 @@ func (p *cfnProvider) Configure(ctx context.Context, req *pulumirpc.ConfigureReq
 
 	var accessKey, secretKey, token string
 
-	if v, ok := varsOrEnv(vars, "aws-native:config:accessKey", "AWS_ACCESS_KEY_ID"); ok {
```

Comment:
This would be a good place for a comment on the `AWS_` vars and that the SDK is already checking them. For future devs debugging issues.

Reply:
Done ✅